### ISSUE TITLE
Improve RN message filter in cplace-cli

### ISSUE THREAD

**Issue by csemrau** [collaborationFactory/cplace-npm-tools#11]
Commit messages starting with `Merge branch 'master' ` or `Merge remote-tracking branch 'origin/master' ` should be ignored.

The generated RN message usually looks like this:

    Merge branch 'master' of https://github.com/collaborationFactory/cplace into feature/PFM-ISSUE-1083-some-feature # Conflicts: #	someFile

    Merge branch 'master' into feature/PFM-ISSUE-1083-some-feature # Conflicts: #	someFile

    Merge remote-tracking branch 'origin/master' into improvement/pfm-issue-1666 

This commit is picked up due to the `PFM-ISSUE-nn` marker in the branch name, but is of no interest to us.

There are some more merges of other branches into feature branches; most of them are to be ignored, but I dare not discard them all without considering more examples.

**Comment by lennarthohl** [comment]
In addition to this description, there is an additional request regarding the markers `ISSUE-` and `US-`. Summary: The intention was that only commit messages marked with `changelog` will be relevant for Release-Notes, because it is a common thing to reference e.g. an issue in a commit without the intention to make this special commit relevant for  the RN's.

The full discussion can be tracked in a featuretalk-platform thread: https://collaborationfactory.slack.com/archives/C4U2ST1MG/p1556543410155400

**Comment by csemrau** [comment]
TL;DR:
* The only commit marker to be used to find new relevant commits is `changelog:`. All other commit markers are to be discarded.
* The messages contained in `messages_*.db` should be treated as if they came from `explicits_*.db`, in that the existence of an annotated commit in that file (for a commit that is part of the diff in question) should lead to the inclusion of the corresponding message into the RN.

The second point is for backwards compatibility, because the new cplace-cli version must handle older releases, which expected commits to be found by the original set of commit markers. It also simplifies usage.

Newly found messages are still added to `messages_*.db`, but existing messages from `explicits_*.db` can then be moved over into the first file without changing the resulting RN messages, and we end up with a single db file.
